src/explore-education-statistics-admin/src/pages/documentation/DocumentationStyle.tsx

```diff
@@ -1181,26 +1282,23 @@ const DocumentationGlossary = () => {
               <p>You can write in full or use the initials.</p>
               <h3 id="plain-english">plain English</h3>
               <p>
-                Lower case plain and upper case English unless in a title: the
-                Plain English Campaign.
+                Do not use formal or long words when easy or short ones will do.
               </p>
               <p>
-                All content on explore ediucation statistics should be written
-                in{' '}
-                <a href="https://www.gov.uk/guidance/content-design/writing-for-gov-uk#plain-english">
-                  plain English
-                </a>
-                . You should also make sure you use language your audience will
-                understand - check which{' '}
-                <a href="https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#words-to-avoid">
-                  words you should avoid
-                </a>
-                .
+                Users have a primary set of vocabulary of 5,000 common words
+                which most use every day so try and use these to get your point
+                across.
+              </p>
+              <p>
+                See
+                <a href="#words-to-avoid">words to avoid</a>.
```

Comment:
links to words to avoid should all read "Words to avoid" (then followed by a full-stop), here we have it without uppercased 'words' please check all instances